plugins/alerts/frontend/public/stylesheets/vue-main.scss

```diff
@@ -0,0 +1,56 @@
+#alerts-view {
```

Comment:
its better if the name of the file is **main.scss** instead of **vue-main.scss**